Description: Fix autoreconf for the minimal audacity source tarball.
 The minimal audacity source tarball does not ship every library in lib-src.
Author: Benjamin Drung <bdrung@debian.org>

--- a/lib-src/Makefile.am
+++ b/lib-src/Makefile.am
@@ -176,7 +176,7 @@ SUBDIRS += lib-widget-extra
 endif
 
 # Note: These optional libraries will not be distributed in the source tarball.
-SUBDIRS += $(OPTIONAL_SUBDIRS)
+#SUBDIRS += $(OPTIONAL_SUBDIRS)
 
 # Overwrite install* targets from automake. We do not want to install anything
 # from the lib-src libraries, because we statically link these libraries.
